BUGS.txt

```diff
@@ -11,6 +11,9 @@ You may also find help at the SDL forums/mailing list:
 
     https://discourse.libsdl.org/
 
-Bug reports are welcome here, but we really appreciate if you use the issue
- tracker, as bugs discussed on the mailing list may be forgotten or missed.
+or on Discord:
+
+    https://discord.com/invite/BwpFGBWsv8
```

Comment:
This server is not operated/owned by any of the SDL team.
Perhaps you should ask for ownership/admin status?

Reply:
I believe Sam and I have "moderator" access on it.

Reply:
We've also been offered admin status, which we can accept anytime we want the power.